lib/pages/home/home_page.dart

```diff
@@ -103,6 +105,7 @@ class HomePage extends StatelessWidget {
                   .toList(),
             ),
           ),
+          NewsFeedCard(),
```

Comment:
Maybe move it above the FAQ? I think it's the kind of thing people would check more frequently.